static/templates/admin/runEnhancementsForm.html

```diff
@@ -0,0 +1,19 @@
+<form id="run_enhancements_form"class="edit_form">
+	<p>
+		All appropriate enhancements will be performed on the selected objects and their children
```

Comment:
This reads a little strangely to me. What is an "appropriate" enhancement? Also, if this meant as a full sentence, it needs to end with a period.

(We could probably just drop this label.)